connect-eh/docker-compose-confluent.yml

```diff
@@ -1,14 +1,16 @@
 ---
-version: '2'
+version: '3'
 services:
   connect:
-    image: confluentinc/cp-kafka-connect:latest
+    image: confluentinc/cp-kafka-connect:4.1.1-2
     hostname: connect
     container_name: connect
     ports:
       - "8083:8083"
+    extra_hosts:
```

Comment:
I don't want to add this part because it only applies to us (working with the emulator). Anyone else wanting to use this doesn't need extra_hosts. That's why I left it in the readme